Rakefile

```
@@ -0,0 +1,126 @@
+require "json"
+
+class Dashboard
+  attr_reader :path
+  attr_accessor :definition
+
+  def initialize(path)
+    @path = path
+  end
+
+  def issues
+    @issues ||= []
+  end
+
+  def validate
+    required_field "metric_keys"
```

```suggestion
    required_field "metric_keys"
    required_field "dashboard"
```

For completeness, to raise an error if the dashboard property is missing entirely.